lib/foreman_inventory_upload/generators/slice.rb

```diff
@@ -37,6 +37,7 @@ def report_host(host)
           @stream.simple_field('account', host.subscription_facet.pools.first.account_number.to_s)
           @stream.simple_field('subscription_manager_id', host.subscription_facet.uuid)
           @stream.simple_field('satellite_id', host.subscription_facet.uuid)
+          @stream.simple_field('satellite_instance_id', Foreman.instance_id)
```

Comment:
It will not be recognized here. Non-canonical fields should be added in `facts` node. See https://github.com/laviro/foreman_inventory_upload/pull/87 for more details.